src/Core/Core.csproj

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build;RunGrunt" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Because the compiled assembly uses embedded resources, would `RunGrunt` need to come before `Build`? And does this extra `Target` need to be incorporated into the `build/Build.msbuild` script?